ui/tests/e2e/specs/project-mapping.spec.ts

```diff
@@ -18,6 +18,11 @@ describe('CSV mapping flow', () => {
       .find('input')
       .type('My project')
 
+    cy.contains('.label', 'Publish graph')
+      .next()
+      .find('input')
+      .type('http://example.com/published-cube')
+
```

Comment:
@martinmaillard what's missing here? it did not seem to find or type in the input field

Reply:
I need to try it out

Reply:
Thanks @martinmaillard